stdlib/http/cookies.pyi

```diff
@@ -49,8 +49,8 @@ class Morsel(dict[str, Any], Generic[_T]):
 
 class BaseCookie(dict[str, Morsel[_T]], Generic[_T]):
     def __init__(self, input: _DataType | None = None) -> None: ...
-    def value_decode(self, val: str) -> _T: ...
-    def value_encode(self, val: _T) -> str: ...
+    def value_decode(self, val: str) -> tuple[_T, str]: ...
+    def value_encode(self, val: _T) -> tuple[str, _T]: ...
```

Comment:
I based the argument positioning by looking at `SimpleCookie`'s implementation:

https://github.com/python/cpython/blob/1ee50e2a78f644d81d341a08562073ad169d8cc7/Lib/http/cookies.py#L607-L612

I think that's what it should look like? Or is it incorrect? I'm getting a bit confused by [the documentation](https://docs.python.org/3/library/http.cookies.html#http.cookies.BaseCookie.value_decode)..

Perhaps I'm just mixing things up regarding en/decode:ing